forest/sycamore/sycamore_main.py

```diff
@@ -1,48 +1,109 @@
 import os
-from .functions import aggregate_surveys_config, aggregate_surveys_no_config
-from .survey_config import survey_submits, survey_submits_no_config
-from .changed_answers import agg_changed_answers_summary
+from typing import Optional, List
 
+from forest.sycamore.functions import (aggregate_surveys_config,
+                                       aggregate_surveys_no_config)
+from forest.sycamore.survey_config import (survey_submits,
+                                           survey_submits_no_config,
+                                           get_all_interventions_dict)
+from forest.sycamore.changed_answers import agg_changed_answers_summary
 
-def survey_stats_main(output_dir, study_dir, beiwe_ids, config_path=None, time_start=None, time_end=None,
-                      study_tz=None):
+
+def survey_stats_main(
+        study_folder: str,
+        output_folder: str,
+        tz_str: Optional[str] = "America/New_York",
```

Comment:
Is this a useful default?

Reply:
I believe around 70% of studies are done in our time zones, so I made this the default. But, perhaps users would expect UTC as default.

Reply:
Since Forest will be released as a library for worldwide use I think it's better to assume UTC as a default.